### ISSUE TITLE
Don't fallback to vi if $EDITOR isn't set

### ISSUE THREAD

**Issue by mirh** [rmarquis/pacaur#581]
Asking the user (like yaourt does) seems way better. 
Especially considering it would also solve the silent skip of the -e option when editor isn't set and vi isn't available.

**Comment by rmarquis** [comment]
Current set up seems perfectly fine to me. Wontfix.

**Comment by mirh** [comment]
> Current set up seems perfectly fine to me.

... Nothing else?
And why that 4.7.x label if invalid?

**Comment by rmarquis** [comment]
> Current set up seems perfectly fine to me.

Yes, nothing else.

**Comment by mirh** [comment]
So if choice is not an option, at least a fix for silent failing?

**Comment by rmarquis** [comment]
Does it not work already? Or did the current check break somehow?

```
[[ ! $editor ]] && [[ ! -f /usr/bin/vi ]] && Note "e" $"${colorW}editor${reset} variable unset"
```

**Comment by mirh** [comment]
I get a line 859: vi: command not found but the program proceeds regardless.

**Comment by rmarquis** [comment]
Fixed in 0f6867bb67d0fd868d3d8c658eea9c20680eff43.

The check wasn't robust enough, and the check with default settings got broken somehow, probably when I cleanup the config options.

**Comment by mirh** [comment]
Ok now it makes sense. 

For as much as I'd still like a more immediate prompt to advance.
